.github/workflows/linter.yml

```
@@ -12,6 +12,17 @@ on:
       - "**"
 
 jobs:
+  buf-lint:
+    name: Protobuf Lint
+    runs-on: ubuntu-latest
```

Let's use 22.04 for consistency
```suggestion
    runs-on: ubuntu-22.04
```